### DISCUSSION TITLE
Using planetscale as Queue Laravel driver return error: Packets out of order.

### DISCUSSION THREAD

**Discussion by taufiqridha** [planetscale/discussion#468] [Q&A] [answered]
Dear All,
I'm trying using PlanetScale as cloud Database for my Laravel Application.
Everything works great except for Queue.
I'm using Database (PlanetScale) as my Queue driver 
QUEUE_CONNECTION=database

Results:
- Jobs entry successfully append into table 'jobs' 
-  But when executing jobs it returns errors and i assume it relates to their connection with PlanetScale.
Here's the log :
Packets out of order. Expected 1 received 2. Packet size=49 at /vendor/laravel/framework/src/Illuminate/Database/Concerns/ManagesTransactions.php:291)

If there's someone has successfully integrated Laravel Queue driver using PlanetScale, please share 
Thanks!

**Comment by orware** [comment, marked as answer]
Hello @taufiqridha,

I would place the Laravel Database Queue option for PlanetScale databases in an "experimental" support status at the moment, mostly related to the following Laravel PR:
https://github.com/laravel/framework/pull/44027

Along with the following Vitess issue:
https://github.com/vitessio/vitess/issues/5249

Since we don't yet have full support for the `FOR UPDATE SKIP LOCKED` usage being utilized within Laravel there, which may result in unexpected behavior.

If you'd like to do a bit more digging into it, I'd be happy to experiment a little further with you more directly over at https://support.planetscale.com/ if needed, even if we may not be able to get past this particular issue right away.

As a more immediate workaround, switching to Laravel's Redis Queue Driver would be another good option to consider.

**Reply by taufiqridha** [reply]
Hai @orware thanks for clear explanation. 
I'll consider using Redis as driver now. though would love if PlanetScale fully support FOR UPDATE SKIP LOCKED.
Thanks !
